DMX.Portal/DMX.Portal.web/Program.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DMX.Portal.web
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();

```

Comment:
Fat arrow a single line method - for instance:

instead of:
```csharp
public static string GetName()
{
    return "Geetika";
}
```

have it say:
```csharp
public static string GetName() => "Geetika";
```
If the return type is too long (longer than 120 characters) then you may do this:
```csharp
public static string GetName() =>
    AVeryLongNamedMethodCall();
```
Mind the indentations.